src/components/Class/ClassCard/index.js

```diff
@@ -183,60 +197,141 @@ function ClassCard({ item, editClass, enroll, style }) {
       dispatch(classActions.dropOutClass(Id));
     });
   };
+
+  const EnrolledAndTimer = () => {
+    const [Timer, setTimer] = useState(TimeLeft(item.start_time));
+    const ONE_MINUTE = 60000; //millisecs
+    setInterval(() => {
+      setTimer(TimeLeft(item.start_time));
+    }, ONE_MINUTE);
+    return (
+      <>
+        {Timer == "joinNow" ? (
+          <a
+            style={{
+              textDecoration: "none",
+            }}
+            href={item.meet_link}
+            target="_blank"
+          >
+            <Button variant="contained" fullWidth>
+              Join Now
+            </Button>
+          </a>
+        ) : (
+          <Button disabled={true} variant="contained">
+            Starts in {Timer}
+          </Button>
+        )}
+      </>
+    );
+  };
   return (
     <>
-      {" "}
       <Card elevation={2} sx={{ p: 4 }} className={classes.card}>
         <Typography
           variant="subtitle1"
           color="primary"
-          className={classes.spacing}
+          sx={{
+            display: "flex",
+            justifyContent: "space-between",
+          }}
         >
           {languageMap[item.type]}
           {item.enrolled && (
             <i className="check-icon check-icon fa fa-check-circle">Enrolled</i>
           )}
+          {rolesList.length == 0 && item.enrolled ? (
+            <MoreVertIcon onClick={handleOpenUserMenu} sx={{ p: 0 }} />
+          ) : (
+            rolesList.length >= 1 &&
+            (item.facilitator.email === user.data.user.email || flag) && (
+              <MoreVertIcon onClick={handleOpenUserMenu} sx={{ p: 0 }} />
+            )
+          )}
```

Comment:
The condition for which to display `MoreVertIcon` is a little complex; I feel as if it should be stored in a descriptive variable.  However, this eliminates the duplication.
```suggestion
          {(
             rolesList.length == 0 && item.enrolled ||
             rolesList.length >= 1 && (item.facilitator.email === user.data.user.email || flag)
           ) && (
             <MoreVertIcon onClick={handleOpenUserMenu} sx={{ p: 0 }} />
            )
          )}
```